### ISSUE TITLE
False negative for properly associated label hidden via sr-only with clip property

### ISSUE THREAD

**Issue by patrickhlauke** [dequelabs/axe-core#1578]
See https://github.com/twbs/bootstrap/issues/28795 and test cases https://gearside.com/labels.html

It appears that Lighthouse's version of axe treats a properly associated label visually hidden using

```
.sr-only {
    border: 0;
    clip: rect(0,0,0,0);
    height: 1px;
    margin: -1px;
    overflow: hidden;
    padding: 0;
    position: absolute;
    width: 1px;
}
```

as not present in the accessibility tree, and as a result flags the associated form input for not having a label. The offending CSS that triggers this is seems to be the `clip: rect(...)` property.

Expectation: unless that particular property does indeed have some compat issues in particular browser/AT combinations (admittedly, I've not checked in all, but a cursory look on Windows with NVDA seemed ok), it should not be flagged as a failure.

Actual: Lighthouse's axe test flags this as an issue https://dequeuniversity.com/rules/axe/3.1/label?application=lighthouse

**Comment by WilcoFiers** [comment]
@patrickhlauke This is a bug that was fixed in axe-core 3.2.0. Lighthouse will have to upgrade it's version of axe-core to fix this.
